rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -251,6 +254,43 @@ def _create_networks(self, network_create_args):
         return lswitches
 
 
+    @atomic.action_timer("ovn_network.of_check_port")
+    def _of_check_ports(self, lports, sandboxes, port_bind_args):
```

Comment:
@hzhou8 I agreed with both of your points; we should use the ```--wait hv``` [1] to measure the latency of binding lport. But I still have two questions:

1. In #97, can we simply add ``` --wait hv``` to all the nb-ctl command so that the time in rally-ovs results reflect the binding time from issuing the command to the OF rules installed on all involved hypervisors? Is this enough?

2. Even with ```--wait hv```, we somehow still want verify the correctness of the flows. As flow tracing is too heave as your mentioned, do you think I can implement it as a asynchronous thread? In other words, besides the main rally-ovs thread, a separate python thread is launched to trace the flow. What do you think? Thanks.

[1] #97